portability-core/src/main/java/org/dataportabilityproject/serviceProviders/instagram/InstagramServiceProvider.java

```diff
@@ -59,12 +65,20 @@ public String getName() {
   }
 
   @Override
-  public OfflineAuthDataGenerator getOfflineAuthDataGenerator(PortableDataType dataType) {
+  public OfflineAuthDataGenerator getOfflineAuthDataGenerator(PortableDataType dataType, ServiceMode serviceMode) {
+    Preconditions.checkArgument(serviceMode == ServiceMode.EXPORT, "IMPORT not supported by Instagram");
+    Preconditions
+        .checkArgument(exportTypes.contains(dataType), "DataType %s not supported by Instagram",
```

Comment:
Let's surround %s in brackets or put it after a semicolon to highlight it is dynamically populated

export [%s] type or export type: %s"

Reply:
Done